example/examples/OpenWindow.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {Component} from 'react';
+import {Button, Switch, StyleSheet, Text, View} from 'react-native';
+
+import WebView from 'react-native-webview';
+
+const HTML = `
+<!DOCTYPE html>\n
+<html>
+  <head>
+    <title>Alerts</title>
```

Comment:
Possibly should be changed to something like "Open Window test"? (though I don't recall offhand if the title is actually ever visible when running the example app..)

Reply:
It doesn't appear in the app, but it is used as Title in browsers' devtools.

I made the fix in https://github.com/react-native-webview/react-native-webview/compare/5e7a71e0ea21bc0d1eeaa492683a306d9fbea576..1b03904b3c44b0e31fa7e7108bae9b30c1af1418